### PR TITLE
Chore: fix tests with beforeEach

### PR DESCRIPTION
**Pull request by weslley543**
## Proposed changes (including videos or screenshots)
- Update test hooks

### CODE REVIEW

File: apps/meteor/tests/e2e/14-setting-permissions.spec.ts [hunk 1]
```diff
@@ -5,22 +5,22 @@ import { validUserInserted } from './utils/mocks/userAndPasswordMock';
 import { Auth, Administration, HomeChannel } from './page-objects';
 
 test.describe('Settings Permissions', () => {
-	let page: Page;
+	let pageTestContext: Page;
```
Comment (souzaramon) [inline apps/meteor/tests/e2e/14-setting-permissions.spec.ts:8]: We should not store pageContext anymore

File: apps/meteor/tests/e2e/16-discussion.spec.ts [hunk 2]
```diff
@@ -5,17 +5,17 @@ import { v4 as uuid } from 'uuid';
 import { Auth, HomeDiscussion } from './page-objects';
 
 test.describe('[Discussion]', () => {
-	let page: Page;
+	let pageTestContext: Page;
```
Comment (souzaramon) [inline apps/meteor/tests/e2e/16-discussion.spec.ts:8]: .

File: apps/meteor/tests/e2e/omnichannel-agents.spec.ts [hunk 3]
```diff
@@ -3,24 +3,24 @@ import { test, expect, Page } from '@playwright/test';
 import { Auth, OmnichannelAgents } from './page-objects';
 
 test.describe('Agents', () => {
-	let page: Page;
+	let pageTestContext: Page;
```
Comment (souzaramon) [inline apps/meteor/tests/e2e/omnichannel-agents.spec.ts:6]: .

File: apps/meteor/tests/e2e/omnichannel-departaments.spec.ts [hunk 4]
```diff
@@ -3,19 +3,19 @@ import { test, Page, expect } from '@playwright/test';
 import { Auth, OmnichannelDepartaments } from './page-objects';
 
 test.describe('Department', () => {
-	let page: Page;
+	let pageTestContext: Page;
```
Comment (souzaramon) [inline apps/meteor/tests/e2e/omnichannel-departaments.spec.ts:6]: .

**Review by souzaramon — DISMISSED** [review]

**Review by ggazzo — APPROVED** [review]

**Pull request merged**